presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -196,14 +196,14 @@
 import static java.util.Collections.emptyList;
 import static java.util.Objects.requireNonNull;
 import static java.util.concurrent.Executors.newCachedThreadPool;
-import static java.util.concurrent.Executors.newSingleThreadScheduledExecutor;
+import static java.util.concurrent.Executors.newScheduledThreadPool;
 
 public class LocalQueryRunner
         implements QueryRunner
 {
     private final Session defaultSession;
     private final ExecutorService executor;
-    private final ScheduledExecutorService transactionCheckExecutor;
+    private final ScheduledExecutorService scheduledExecutor;
```

Comment:
Rename `executor` to `notificationExecutor`. Rename `scheduledExecutor` to `yieldExecutor`